plane/source/docs/guide-four-channel-plane.rst

```diff
@@ -78,6 +80,8 @@ If the any of the control surfaces do not respond correctly, reverse the
 output by changing the corresponding SERVOn_REVERSED setting (from 0 to 1, 
 or from 1 to 0).
 
+The output can be reversed in Mission Planner on the Setup -> Servo Output page, just check the box for the channel that needs to be reversed. This will set the correct SERVOn_REVERSED value under the covers.
```

Comment:
```suggestion
The output can be reversed in Mission Planner on the SETUP/Mandatory Hardware/Servo Output page, just check the box for the channel that needs to be reversed. This will set the output's SERVOn_REVERSED parameter.
```